core/markdown_api.php

```diff
@@ -0,0 +1,106 @@
+<?php
+# MantisBT - A PHP based bugtracking system
+
+# MantisBT is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 2 of the License, or
+# (at your option) any later version.
+#
+# MantisBT is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with MantisBT.  If not, see <http://www.gnu.org/licenses/>.
+
+/**
+ * Markdown API
+ *
+ * @package CoreAPI
+ * @subpackage MarkdownAPI
```

Comment:
This api is unncessary as it is now
If you need "firendly" methods and need to track a sigle object instance, then add static methods
MantisMarkdown::markdown_text
MantisMarkdown::markdown_line

There you can deal with the single instance internally to the class, no need for the external code to keep track of a global variable and markdown_init()